static/styles/homepage/login.scss

```diff
@@ -23,5 +23,49 @@
 	}
     .login-providers {
         display:none;
-    }
+	}
+}
+
+.login-failed-container{
+	margin-top: 30px;
+	max-width: 900px;
```

Comment:
Maybe we should place the div in the middle, otherwise I think it looks weird.
![Opera Momentaufnahme_2020-05-11_141516_localhost](https://user-images.githubusercontent.com/36112357/81560532-dddcd680-9391-11ea-9f67-b1e4781e3aca.png)

Reply:
Maybe we can make a collapsible out of it? Or each bullet point as a collapsible?

Reply:
would be an option..